### ISSUE TITLE
Graph color option. (Colorblind support)

### ISSUE THREAD

**Issue by Pixel-Null** [numworks/epsilon#931]
Would it be possible to add a color option in the function app for each different function?
So you'd be able to select colors for a function.
with a color menu added to the function options or the use of arrows in a specific key combination to cycle through the available colors?

I'm personally colorblind and i have a hard time seeing the difference between them, and having to add and remove functions until i get a combination of colors i can clearly see the difference between can be quite the hassle.

Thanks in advance!

**Comment by boricj** [comment]
Do you have any other issues with other screens? I've raised the question of accessibility before (#597), but I think this is the first time we actually have a user that is affected by this on hand.

**Comment by PnDevs** [comment]
I do yes, on every screen (red and green colorblind), some of them have colorblind filters built in but honestly it's just an issue in graphs at the moment so i wouldn't bother making a filter for the entire calculator.

**Comment by DSchndr** [comment]
Maybe making an colorblind palette in escher, which can be enabled from the settings menu could be the solution for this problem.
Because then you only have to change a couple of values that are used globally.

When i have time i'll experiment a little bit with the idea and ask one from my class (who is r/g colorblind) if it is better.

**Comment by boricj** [comment]
The NumWorks team has said that they aren't planning to introduce themes support at the moment. I came up #947 recently and its ability to invert colors and perform per-channel gamma correction might help here. A pre-compiled firmware is available at https://tiplanet.org/forum/viewtopic.php?f=97&t=22485 if you want to try it out.

Otherwise, changing the data color palette would be the way to go (escher/include/escher/palette.h).

**Comment by DSchndr** [comment]
Yeah, i also know about the theme support, but i think that it would be awesome to add some accessibility options to epsilon to make the calculator more interesting.
Per-channel gamma correction would be really nice, because then we don't need to correct images if there are any that need correction.

As for the graphing issue, i've replaced the palette and it seems to work, but i still need to ask how much better it looks.

```
//Paul Tol colorblind pallette (https://davidmathlogic.com/colorblind)
  constexpr static KDColor CBPurple = KDColor::RGB24(0x332288);
  constexpr static KDColor CBGreen = KDColor::RGB24(0x117733);
  constexpr static KDColor CBOGreen = KDColor::RGB24(0x44AA99);
  constexpr static KDColor CBBlue = KDColor::RGB24(0x88CCEE);
  constexpr static KDColor CBYellow = KDColor::RGB24(0xDDCC77);
  constexpr static KDColor CBRed1 = KDColor::RGB24(0xCC6677);
  constexpr static KDColor CBRed2 = KDColor::RGB24(0xAA4499);
  constexpr static KDColor CBRed3 = KDColor::RGB24(0x882255);

  constexpr static KDColor DataColor[] = {CBPurple, CBGreen, CBOGreen, CBBlue, CBYellow, CBRed1, CBRed2, CBRed3};
```
![Bildschirmfoto vom 2019-04-11 21-55-20](https://user-images.githubusercontent.com/19175445/55991698-24519480-5cab-11e9-9bab-9936227a1f32.png)

**Comment by artaxxx** [comment]
Fixed in 19.0.0
